sdk/eventhub/eventhubs-checkpointstore-blob/src/blobCheckpointStore.ts

```diff
@@ -107,12 +107,21 @@ export class BlobCheckpointStore implements CheckpointStore {
           `LastModifiedTime: ${ownership.lastModifiedTimeInMs}, ETag: ${ownership.etag}`
         );
       } catch (err) {
-        // NOTE: there is some ordinary contention that can occur as different consumers battle over
-        // ownership. So the catching (and _only_ logging, not rethrowing) of this error is intentional.
+        const restError = err as RestError;
+
+        if (restError.statusCode === 412) {
+          // etag failures (precondition not met) aren't fatal errors. They happen
```

Comment:
Can you add a log statement when this happens?

Reply:
I was thinking about that too, mostly because of the ownership-flapping issue you were looking at. 

I'll add that with a little verbiage to say that the error is not fatal so nobody is alarmed if they see it. (also, mark it as 'info' or whatever the equivalent is in our logging)

Reply:
Done. :)